### PR TITLE
Add 猫耳

### PR DESCRIPTION
**Pull request by acid-chicken**
image

### CODE REVIEW

File: src/client/app/common/views/components/avatar.vue [hunk 1]
```diff
@@ -30,6 +38,9 @@ export default Vue.extend({
 		lightmode(): boolean {
 			return this.$store.state.device.lightmode;
 		},
+		cat(): boolean {
+			return this.user.isCat && this.$store.state.settings.circleIcons;
+		}
```
Comment (syuilo) [inline src/client/app/common/views/components/avatar.vue]: コロン要りそう

Comment (acid-chicken) [reply src/client/app/common/views/components/avatar.vue]: @syuilo I've fixed, thanks!

File: src/client/app/common/views/components/avatar.vue [hunk 2]
```diff
@@ -1,8 +1,16 @@
 <template>
-	<span class="mk-avatar" :title="user | acct" :style="style" v-if="disableLink && !disablePreview" v-user-preview="user.id" @click="onClick"></span>
-	<span class="mk-avatar" :title="user | acct" :style="style" v-else-if="disableLink && disablePreview" @click="onClick"></span>
-	<router-link class="mk-avatar" :to="user | userPage" :title="user | acct" :target="target" :style="style" v-else-if="!disableLink && !disablePreview" v-user-preview="user.id"></router-link>
-	<router-link class="mk-avatar" :to="user | userPage" :title="user | acct" :target="target" :style="style" v-else-if="!disableLink && disablePreview"></router-link>
+	<span :class="{ mk-avatar: true, cat: cat }" :title="user | acct" v-if="disableLink && !disablePreview" v-user-preview="user.id" @click="onClick">
```
Comment (AyaMorisawa) [inline src/client/app/common/views/components/avatar.vue]: `:class="{ mk-avatar: true, cat: cat }"`
must be
`class="mk-avatar" :class="{ cat }"`

**Review by AyaMorisawa — CHANGES_REQUESTED** [review]

File: src/client/app/common/views/components/avatar.vue (hunk 2 above)
Comment (acid-chicken) [reply src/client/app/common/views/components/avatar.vue]: @AyaMorisawa Done, thanks!

**Review by AyaMorisawa — APPROVED** [review]

**Review by AyaMorisawa — COMMENTED** [review]
To support dark mode:

1.  Insert this code before `</style>`.
```
.mk-avatar[data-darkmode]
	root(true)

.mk-avatar:not([data-darkmode])
	root(false)
```

2. Replace `.mk-avatar` with `root(isDark)`

3. Replace `border solid 4px #202224` with `border solid 4px isDark ? #e0eefd : #202224`

Then remove `white()` function to clean the code.

----
I will merge this PR and fix these things in `syuilo/master`. Please follow these steps in future contributions! Thank you! :+1:

**Pull request merged**